content/en/docs/components/model-registry/getting-started.md

```diff
@@ -0,0 +1,169 @@
++++
+title = "Getting started"
+description = "Getting started with Model Registry using examples"
+weight = 30
+
++++
+
+This guide shows how to get started with Model Registry and run a few examples using the
+command line or Python clients.
+
+At this time, there is not yet an UI for Model Registry; therefore, this documentation focus on backend services and APIs.
+
+For an overview of the logical model of model registry, check the
+[Model Registry logical model](https://github.com/kubeflow/model-registry/blob/main/docs/logical_model.md).
+The logical model is exposed via the Model Registry [REST API](https://editor.swagger.io/?url=https://raw.githubusercontent.com/kubeflow/model-registry/main/api/openapi/model-registry.yaml).
+
+## Prerequisites
+
+To follow along the examples in this guide, you will need a Kubeflow installation and the Model Registry installed:
+
+- Kubeflow [installed](docs/started/installing-kubeflow/)
```

Comment:
Can model registry be deployed without Kubeflow ?

Reply:
Yes. Do you want me to expand in this section how Model Registry can be installed without Kubeflow just on K8s?

Reply:
@tarilabs I think, that would be nice given that we list Model Registry in the Kubeflow Standalone Components table: https://github.com/kubeflow/website/pull/3724

Reply:
the link is not working for me

Reply:
> the link is not working for me

this was solved with: https://github.com/kubeflow/website/commit/d9c3cd6ed939dc22de87671489be976f10756b82

> Can model registry be deployed without Kubeflow ?

tracked in: https://github.com/kubeflow/model-registry/issues/109
proposed impl with: https://github.com/kubeflow/website/pull/3740

marking as resolved this thread accordingly.